backend/Watchdog.Core/Watchdog.Core.DAL/Context/EntityConfigurations/ApplicationConfig.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Watchdog.Core.DAL.Entities;
+
+namespace Watchdog.Core.DAL.Context.EntityConfigurations
+{
+    public class ApplicationConfig : IEntityTypeConfiguration<Application>
+    {
+        public void Configure(EntityTypeBuilder<Application> builder)
+        {
+            builder.Property(e => e.CreatedBy)
+                  .IsRequired();
```

Comment:
You don't need to use IsRequired for fields that cannot be null.
Please fix it in all places with the same logic

Reply:
I applied your recommendations to specified fields.